backend-go/api/codepair/v1/models/errors.go

```
@@ -0,0 +1,11 @@
+package models
+
+import "errors"
+
+func RequiredFieldError(field string) error {
+	return errors.New(field + " is required")
+}
+
+func MinLengthError(field string, min int) error {
```

golangci-lint (1.62.2)

[warning] redefines-builtin-id: redefinition of the built-in function min

(revive)